docs/site/Controller-generator.md

```diff
@@ -69,6 +69,20 @@ to select:
 - The repository for this model that provides datasource connectivity
 - The REST path name to host the endpoints on
 
+The prompts that list out the models and repositories to choose from to build
+the controller with are chosen from the existing model/repository files on disc.
+From the LoopBack 4 project that the CLI is run, the CLI tool will search for
+file names ending with the following strings under `src/models` and
+`src/repositories`: `model.ts`/`model.js` and `repository.ts`/`repository.ts`.
```

Comment:
I find it difficult to understand what does this sentence mean. How about using a wildcard approach instead?

```markdown
The CLI tool will search for the following files in the LoopBack 4 project it runs in:
 - `src/repositories/*.repository.ts` and 
 - `src/repositores/*.repository.js` to find model repositories
 - `src/models/*.model.ts` and 
 - `src/models/*.model.js` to find model classes
```